src/5-ipfs-weekly-1/index.md

```diff
@@ -0,0 +1,118 @@
+---
+date: 2016-01-11
+id: 5-ipfs-weekly-1
+template: tmpl/layouts/post.html
+baseurl: ..
+breadcrumbs:
+  - {name: "3-ipfs-weekly-1", link: "./" }
```

Comment:
this number should match the one in the `id:` also check that 5 is the right number (im not sure, there one post there that's a draft and not published-- can bump its number up for this one, or whatever)